falcon/api_helpers.py

```diff
@@ -0,0 +1,7 @@
+from .app_helpers import * # NOQA
```

Comment:
Nit: Should be two spaces before the `#`

Reply:
nit; using a formatter would take care of it :P